src/main/scala/cognite/spark/v1/AssetsRelation.scala

```diff
@@ -94,19 +99,64 @@ class AssetsRelation(config: RelationConfig)(val sqlContext: SQLContext)
   }
 
   override def getFromRowsAndCreate(rows: Seq[Row], doUpsert: Boolean = true): IO[Unit] = {
-    val assets = rows.map(fromRow[AssetCreate](_))
+    val assetsUpserts = rows.map(fromRow[AssetsUpsertSchema](_))
+    val assets = assetsUpserts.map(assetUpsertSchemaToAssetCreate)
```

Comment:
to make sure we skip labels in upsert (maybe for now). labels are available for create/read.